openwisp_monitoring/device/settings.py

```diff
@@ -1,38 +1,55 @@
 from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
 
+
+def get_or_create_critical_device_metrics():
```

Comment:
Name it just `get_critical_device_metrics`, you can also remove the comment in the following line because it doesn't provide much value